dit/pid/__init__.py

```diff
@@ -11,6 +11,8 @@
 from .idep import PID_dep
 from .idownarrow import PID_downarrow
 from .ipm import PID_PM
+from .hcs import PED_CS
+from . import distributions
```

Comment:
Can you change this to:

    from .distributions import bivariates, trivariates